src/components/RemoveDatasets.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import SelectableDatasets from "../pages/dar_application/SelectableDatasets";
+import {Dataset} from "src/types/model";
+
+interface RemoveDatasetsProps {
+  datasets: Dataset[];
+  readOnlyMode: boolean;
+}
+
+interface FormStateInterface {
+    datasetIds: number[];
+}
+
+export default function RemoveDatasets(_props: RemoveDatasetsProps): React.JSX.Element {
```

Comment:
I'm confused on the naming here - this component is showing you `SelectableDatasets`. I'm also confused on the need to wrap this in a parent component.